packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -332,6 +330,8 @@ where
     ) -> Result<(VersionId, OntologyElementMetadata), UpdateError>
     where
         T: OntologyDatabaseType<Representation: Send> + Send + Sync,
+        T::WithMetadata: PostgresRecord + Send,
+        for<'p> <T::WithMetadata as Record>::QueryPath<'p>: OntologyQueryPath + Send + Sync,
```

Comment:
These bounds are required for `Read<T::WithMetadata>`

Reply:
I had a quick attempt locally and couldn't find an easy way to do it either, I do wonder if we have too many traits here, it does seem overly complicated

Reply:
This is mainly because we have a generic implementation for `Read<T::WithMetadata>` instead `impl Read<DataTypeWithMetadata>` etc.. I expect this to be simplified in a not-too-distant future.